File: src/test/java/ru/AlexeyFedechkin/znatoki/StudyRussianBot/WordManagerTest.java
package ru.AlexeyFedechkin.znatoki.StudyRussianBot;

import org.junit.Test;

import java.io.IOException;

public class WordManagerTest {

    @Test
    public void init() throws IOException {
        WordManager wordManager = new WordManager();
        wordManager.init();
    }
}

File: src/test/java/ru/centralhardware/znatoki/studyRussianBot/WordManagerTest.kt
package ru.centralhardware.znatoki.studyRussianBot

import junit.framework.Assert.assertEquals
import org.junit.Test

class WordManagerTest {

    @Test
    fun init() {
        WordManager.init()
    }

    @Test
    fun word() {
        WordManager.init()
        val word = WordManager.rules[0].words[0]
        assertEquals(word.wrightName, "жир" )
        assertEquals(word.name, "ж_р")
        assertEquals(word.section, "r1")
        assertEquals(word.answer, "и")
    }

}